### PR TITLE
Update modulepath on Hera following rocky8 upgrade. Use  bufr 11.7.0 instead of 12.0.1

### LINKED ISSUE #18: Hera Rocky 8 Migration
Hera is undergoing migration to Rocky 8.
The utilities under this project will need to be built and tested on Hera with Rocky 8

### PR DESCRIPTION
**Pull request by aerorahul**
Closes #18 
bufr 12.0.1 is not available on Hera/Rocky 8.

### CODE REVIEW

File: modulefiles/fit2obs_hera.intel.lua [hunk 1]
```diff
@@ -2,7 +2,8 @@ help([[
 Build environment for fit2obs on Hera
 ]])
 
-prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
+--prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/gsi-addon-dev-rocky8/install/modulefiles/Core")
```
Comment (DavidHuber-NOAA) [inline modulefiles/fit2obs_hera.intel.lua:5]: Just a comment: BUFR 12.0.1 is available in the unified-env-rocky8 build.  However, I think pointing to the gsi-addon environment is the correct way to go so as to maintain the same modules as the global workflow.

File: modulefiles/fit2obs_hera.intel.lua [hunk 2]
```diff
@@ -2,7 +2,8 @@ help([[
 Build environment for fit2obs on Hera
 ]])
 
-prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
+--prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
```
Comment (KateFriedman-NOAA) [inline modulefiles/fit2obs_hera.intel.lua]: Why leave the old `MODULEPATH` here?

**Review by KateFriedman-NOAA — CHANGES_REQUESTED** [review]

File: modulefiles/fit2obs_hera.intel.lua (hunk 2 above)
Comment (aerorahul) [reply modulefiles/fit2obs_hera.intel.lua]: good catch.  Will remove once testing confirms this is good.

**Review by DavidHuber-NOAA — APPROVED** [review]

File: modulefiles/fit2obs_hera.intel.lua (hunk 2 above)
Comment (aerorahul) [inline modulefiles/fit2obs_hera.intel.lua]:
```suggestion
```
remove commented line

File: modulefiles/fit2obs_hera.intel.lua (hunk 1 above)
Comment (aerorahul) [reply modulefiles/fit2obs_hera.intel.lua:5]: ok. thanks for the info.

**Review by KateFriedman-NOAA — APPROVED** [review]
Looks good, thanks @aerorahul !

**Pull request merged**